app/components/language-picker/component.js

```diff
@@ -1,40 +1,57 @@
 import Ember from 'ember';
-
+import config from 'ember-get-config';
 import countries from './countries';
 
-
 export default Ember.Component.extend({
     i18n: Ember.inject.service(),
     countries: countries,
-    languages: Ember.computed('countries', function() {
-      var langs = [];
-      this.get('countries').forEach((country) => {
-        country.languages.forEach((lang) => {
-          langs.push({
-            countryCode: country.code,
-            countryName: country.name,
-            name: lang.name,
-            code: lang.code
-          });
-        });
-      });
-      langs.sort(function(a, b) {
-        if (a.countryName > b.countryName) {
-          return 1;
-        }
-        if (a.countryName < b.countryName) {
-          return -1;
+    languages: Ember.computed('countries', 'i18n.locales', function () {
+        const isProduction = config.environment === 'production';
```

Comment:
`--environment` is a flag that controls the build process in ember. We are trying to avoid overloading the meaning of build flags with custom "which server am I" settings- site and build config should not be intertwined.

(there was a recent PR that took out most of the places where we did this)

For now, we are controlling testing-mode settings using the `featureFlags` section of config/environment.js. Can you please move this option there, or to a similar place as appropriate?